.github/workflows/ci.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
     runs-on: ${{ matrix.platform }}
     env:
       GOPATH: /home/runner/work/kpt-functions-sdk/go
```

Comment:
I think we no longer need to set GOPATH with mod:

https://github.com/actions/setup-go

Reply:
We still use GOPATH for linting.